pressbooks/pressbooks
# chore: remove network checklist

Partial fix pressbooks/pressbooks-network-analytics#366

This PR removes the Network Checklist from the Dashboard

**How to test**

1. Add the following environment variables to your `.env` file
```env
PB_CHECKLIST_BOOKING_URL=https://link-to-calendly.com
PB_CHECKLIST_NETWORK_CREATION_MONTHS_AGO="-10 years"
PB_CHECKLIST_ONBOARDING_SURVEY=https://link-to-the-survey.com
```
2. With the `dev` branch checked out, visit the https://pressbooks.test/wp-admin/network/index.php?page=pb_network_page
3. You should see the Network Checklist as usual
4. Check this branch out and reload the page
5. You should not see the Network Checklist anymore

>[!NOTE]
>Other behaviours should continue to work as expected
